.github/workflows/iroha2-ci-image.yml

```diff
@@ -9,7 +9,9 @@ on:
 
 jobs:
   dockerhub:
-    runs-on: ubuntu-latest
```

Comment:
For some jobs that are not really heavy and sensitive to execution time, no reason to use paid runners. I would revert to `ubuntu-latest`